### ISSUE TITLE
Demo?

### ISSUE THREAD

**Issue by Zireael07** [photopea/UNN.js#8]
I'm not sure how to use this otherwise?

Does it have depthwise convolution? How would I add a new layer type if it doesn't?

**Comment by TrevorBlythe** [comment]
No depthwise conv but I can just add it I didn't know that was a thing.

**Comment by photopea** [comment]
I have never heard of the depthwise convolution either. It should be easy to implement. But are you sure this library (running on a single CPU thread) will be sufficient for your AI needs?

**Comment by Zireael07** [comment]
Depthwise convolution is what MobileNet uses. I do not know the details.

(You claim that this is faster than the other js libraries and comparable with tensorflow js, in the README)

**Comment by photopea** [comment]
Yes, it should be faster, but we dont have this type of convolution in UNN.js. Does it work well in Tensorflow.js in your case?

**Comment by Zireael07** [comment]
Yes

**Comment by TrevorBlythe** [comment]
So is depthwise conv just a conv over a single specific depth ignoring
others? I dont understand

**Comment by Zireael07** [comment]
I think yes, Googling throws up some scientific papers on the topic
